tests/backtest/test_trade_request.py

```diff
@@ -0,0 +1,37 @@
+"""Test the Trade Request object."""
+
+import pytest
+
+from coin_test.backtest import TradeRequest
+
+symbol = "BTC"
+side = "buy"
+type_ = "market"
+notional = 10  # buy $10 worth of Bitcoin
+qty = 2  # buy 2 Bitcoin
```

Comment:
Global variables -> locals of fixtures, see above.

Reply:
Done